src/app/components/dashboard/educational-program/educational-program.component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-educational-program',
+  templateUrl: './educational-program.component.html',
+  styleUrls: ['./educational-program.component.scss']
+})
+export class EducationalProgramComponent implements OnInit {
+  programs = ['All', '.NET', 'JS', 'BA']
```

Comment:
Lets move variables like this to some "global" place, like: /global/constants.ts